Drivers/rc_receiver/Inc/rcreceiver_datatypes.h

```diff
@@ -2,6 +2,7 @@
 #define RC_RECEIVER_DATA_TYPES_H
 
 #include <cstdint>
+#include <iostream>
```

Comment:
I'm surprised this is the header that is required for your change. cppreference.com suggests that `std::copy` is from \<algorithm\>